guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Package Guidelines
+description: Guidelines for Pangeo packages to follow
+---
+
+Our vision for the Pangeo project is an ecosystem of mutually compatible
+Geoscience python packages which follow open-source best practices.
+These practices are well established across the scientific python community.
+
+
+### General Best Practices for Open Source
+
+1. Open source license
+1. Community guidelines (provide example)
+1. Version control of source code (for example, on [github](http://github.org))
+1. Thorough test coverage and continuous integration of testing
+1. Comprehensive Documentation
+
+
+### Best Practices for Pangeo Projects
+
+To address the needs of geoscience researchers, we have developed some
+additional recommendations.
+
+1. _Solve a general problem:_ packages should solve a general problem
+that is encountered by broad groups of researchers.
+1. _Clearly defined scope:_ packages should have a clear and relatively
+narrow scope.
+1. _No duplication:_ developers should try to leverage existing
+packages as much as possible to avoid duplication of effort.
+1. _Consume and Produce XArray Objects:_ XArray data structures facilitate
+mutual interoperability between packages.
+1. _Operate Lazily:_ packages should avoid explicitly triggering computation
+on [Dask](http://dask.pydata.org/en/latest/array.html) objects.
+
+
+### Why XArray and Dask?
+
+The Pangeo project strongly encourages the use of XArray data structures
+wherever possible.
+XArray Dataset and DataArrays contain multidimensional
+numeric array data and also the metadata describing the data's coordinates,
+labels, units, and other relevant attributes.
+XArray makes it easy to keep this important metadata together with the raw data;
+applications can then take advantage of the metadata to perform calculations
```

Comment:
There's a potential role for a simple library of helper functions/decorators that could be used to make metadata preservation simpler and more consisted when operating on xarray objects - such as unit conversions, appending to history, documenting library versions used, etc. All pangeo packages could import this simple library so that we're doing the same metadata preservation operations.